dcos_installer/action_lib.py

```diff
@@ -379,7 +379,24 @@ def _add_prereqs_script(chain):
 distro="$(source /etc/os-release && echo "${ID}")"
 if [[ "${distro}" == 'coreos' ]]; then
   echo "Distro: CoreOS"
-  echo "CoreOS includes all prerequisites by default." >&2
+  sudo systemctl is-enabled systemd-networkd > /dev/null 2>&1
+  networkd=$(echo $?)
+
+  if [ "x$networkd" == "x0" ]; then
```

Comment:
What’s with the adding x in the comparison? Seems unnecessary.

Reply:
I found this idiom followed quite a bit in bash. The reason being that if `$networkd` is empty the interpreter would crash (error out). The `x` allows the above syntax to treat the variable explicitly as a string so an empty $networkd would be "x" == "x0" which is still a valid comparison.

Reply:
I don't think this is true when you're surrounding the variable with quotes. The following will not crash the interpreter:
```
if [ "" == "0" ]; then
```

Do you have an example of when it might fail?

Reply:
Let me try this. Could be a lack of understanding on my part.

Reply:
yeah you are right, that seems to work. Will remove the `x`.